### PR TITLE
G20-728: improve side panel

### PR DESCRIPTION
**Pull request by mpereiraesaa**

### CODE REVIEW

File: src/annotator/config/settings.js [hunk 1]
```diff
@@ -149,6 +149,14 @@ function settingsFrom(window_) {
     return jsonConfigs.query || queryFromURL();
   }
 
+  function getMetaValueSidebarDefaultExpanded() {
+      const annotationSidebarDefaultExpanded = window_.document.querySelector('meta[name="annotationSidebarDefaultExpanded"]').content;
+      if (annotationSidebarDefaultExpanded === 'true') {
```
Comment (asgeir-inspera) [inline src/annotator/config/settings.js]: You can simplify this to `return annotationSidebarDefaultExpanded === 'true';`

File: src/annotator/config/settings.js [hunk 2]
```diff
@@ -149,6 +149,14 @@ function settingsFrom(window_) {
     return jsonConfigs.query || queryFromURL();
   }
 
+  function getMetaValueSidebarDefaultExpanded() {
```
Comment (asgeir-inspera) [inline src/annotator/config/settings.js]: I would just call it "getSidebarDefaultExpanded". Th fact that it comes from a meta tag is not really important to the outside.

File: src/annotator/config/settings.js [hunk 3]
```diff
@@ -189,6 +197,7 @@ function settingsFrom(window_) {
       return query();
     },
     hostPageSetting: hostPageSetting,
+    getMetaValueSidebarDefaultExpanded: getMetaValueSidebarDefaultExpanded,
```
Comment (asgeir-inspera) [inline src/annotator/config/settings.js]: Where is this used?

Comment (mpereiraesaa) [reply src/annotator/config/settings.js]: it's exported to be used in the src/annotator/config/index.js to get the current expanded value.

**Pull request merged**